torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -408,6 +417,25 @@ struct SourceImporterImpl : public Resolver,
                 for (const auto& buffer : buffer_list) {
                   buffer_names.insert(StringLiteral(buffer).text());
                 }
+              } else if (name == "__forward_pre_hooks__") {
+                TORCH_INTERNAL_ASSERT(
+                    is_module,
+                    "Forward pre hooks only exist on modules at the moment");
+                const auto pre_hook_list =
+                    ListLiteral(assign.rhs().get()).inputs();
+                for (const auto& pre_hook : pre_hook_list) {
+                  pre_hook_names.insert(StringLiteral(pre_hook).text());
```

Comment:
Consider refactoring `StringLiteral(pre_hook).text()` to a separate variable to avoid repeating it.